src/components/QuestList/index.tsx

```diff
@@ -0,0 +1,163 @@
+          <div key={quest.id} className={styles.card}>
+            <div className={styles.firstRow}>
+              <p className={classNames('title', styles.days)}>
+                0<span className={styles.day}>/ 7 days</span>
+              </p>
+              <div className={styles.badges}>
+                <p className={styles.badgesSpace}>
+                  <Sticker styleType="tertiary" variant="outlined">
+                    <Refresh className={styles.badgeIcon} />
+                    Weekly
+                  </Sticker>
+                  <Sticker styleType="tertiary" variant="outlined">
+                    2x
+                  </Sticker>
+                </p>
+
+                <p className={styles.resetTime}>
+                  <ClockNew className={styles.clockIcon} />
+                  <span className={styles.time}>
+                    Day resets: 23:43:03 (UTC)
+                  </span>
+                </p>
+              </div>
+            </div>
+            <div className={styles.secondRow}>
+              <p className={styles.questType}>
+                <Lightning className={styles.lightningIcon} />
+                Play Streak
+              </p>
+              <p className={styles.rewardType}>+200 G7C</p>
+            </div>
+          </div>
+        ))}
+
+        {/* Active Quests */}
+        <div className={styles.activeQuests}>
+          <p className={styles.activeQuestsTitle}>
+            <QuestIcon className={styles.questIcon} />
+            Active
+          </p>
+          <Sticker styleType="secondary" variant="filledStrong">
+            Label
+          </Sticker>
+        </div>
+        {activeQuests.map((quest) => (
+          <div key={quest.id} className={styles.card}>
+            <div className={styles.firstRow}>
+              <p className={classNames('title', styles.days)}>
+                0<span className={styles.day}>/ 7 days</span>
+              </p>
+              <div className={styles.badges}>
+                <p className={styles.badgesSpace}>
+                  <Sticker styleType="tertiary" variant="outlined">
+                    <Refresh className={styles.badgeIcon} />
+                    Weekly
+                  </Sticker>
+                  <Sticker styleType="tertiary" variant="outlined">
+                    2x
+                  </Sticker>
+                </p>
+                <p className={styles.resetTime}>
+                  <ClockNew className={styles.clockIcon} />
+                  <span className={styles.time}>
+                    {' '}
+                    Day resets:23:43:03 (UTC)
+                  </span>
+                </p>
+              </div>
+            </div>
+            <div className={styles.secondRow}>
+              <p className={classNames('body', styles.questType)}>
+                <Lightning className={styles.lightningIcon} />
+                Play Streak
+              </p>
+              <p className={styles.rewardType}>+200 G7C</p>
```

Comment:
it might be best if we accepted a `React.ReactElement` as a prop and then showed it here